core-bundle/src/Resources/contao/library/Contao/Date.php

```diff
@@ -638,7 +638,7 @@ protected static function resolveCustomModifiers($strDate)
 
 		foreach ($chunks as $chunk)
 		{
-			list($index, $flag) = explode('::', $chunk);
+			list($index, $flag) = array_pad(explode('::', $chunk), 2, null);
```

Comment:
If I understand correctly, we should probably check for `::` in the `$chunk` and if none is present just add an early-return (add `$chunk` to `$strReturn` and `continue`)?

Reply:
I pushed a new commit. But is this really better? There is code duplication with the default case of the switch now. I guess it’s a matter of taste.

Reply:
Solved differently based on review by @leofeyer.